OracleCoherence/README.md

```diff
@@ -1,115 +1,193 @@
+    sh buildDockerImage.sh -s
+    ```
 
     Or to run the Quick installer use:
 
-        $ sh buildDockerImage.sh -q
+    ```shell script
+    sh buildDockerImage.sh -q
+    ```
 
-    If you are using a different version of Coherence than 12.2.1.3.0 then you can use the `-v` parameter to specify a version. For example if you are using `12.2.1.2.0` you would run:
+    If you are using a different version of Coherence than 12.2.1.3.0 then you can use the `-v`
+    parameter to specify a version. For example if you are using 12.2.1.2.0 you would run:
 
-        $ sh buildDockerImage.sh -v 12.2.1.2.0
+    ```shell script
+    sh buildDockerImage.sh -v 12.2.1.2.0
+    ```
 
-5. The resulting image file will be called oracle/coherence:${version}-${distribution}, for example if the Standalone installer is used the image will be `oracle/coherence:12.2.1.3.0-standalone`
+5. The resulting image file will be called oracle/coherence:${version}-${distribution}, for example
+   if the Standalone installer is used the image will be `oracle/coherence:12.2.1.3.0-standalone`
 
-6. The image is built with a shell script as its ENTRYPOINT that allows the image to be run using the normal Docker run command. See the [Image Usage](00.imageusage) documentation.
+6. The image is built with a shell script as its ENTRYPOINT that allows the image to be run using
+   the normal Docker run command. See the [Image Usage](00.imageusage) documentation.
 
 ### For Coherence 12.2.1.3.2 follow this process
 
 1. Checkout the GitHub Oracle Docker Images repository
 
-	`$ git clone git@github.com:oracle/docker-images.git`
+    ```shell script
+    git clone git@github.com:oracle/docker-images.git
+    ```
 
-2. Go to the **OracleCoherence/dockerfiles/12.2.1.3.2** folder
+2. Go to the `OracleCoherence/dockerfiles/12.2.1.3.2` folder
 
-        $ cd OracleCoherence/dockerfiles/12.2.1.3.2
+    ```shell script
+    cd OracleCoherence/dockerfiles/12.2.1.3.2
+    ```
 
-3. [Download](https://www.oracle.com/middleware/technologies/coherence-downloads.html) and drop the Coherence distribution file of your choice into this folder. The build script supports either building an image from either the Standalone Installer, **fmw_12.2.1.3.0_coherence_Disk1_1of1.zip** or the Quick Installer **fmw_12.2.1.3.0_coherence_quick_Disk1_1of1.zip**
+3. [Download](https://www.oracle.com/middleware/technologies/coherence-downloads.html) and drop the
+   Coherence distribution file of your choice into this folder. The build script supports either
+   building an image from either the Standalone Installer, `fmw_12.2.1.3.0_coherence_Disk1_1of1.zip`
+   or the Quick Installer `fmw_12.2.1.3.0_coherence_quick_Disk1_1of1.zip`
 
-4. Download [Coherence 12.2.1.3.2 cumulative patch file](https://updates.oracle.com/Orion/PatchDetails/process_form?patch_num=29204496) into this folder
+4. Download [Coherence 12.2.1.3.2 cumulative patch file](https://updates.oracle.com/Orion/PatchDetails/process_form?patch_num=29204496)
```

Comment:
If this patch is only accessible to customers, you need to provide a notice that this image can only  be built by customers with an active support subscription.